### PR TITLE
update packages 24/07/2019

### PR DESCRIPTION
**Pull request by michalrak**
- Update packages to resolve security vulnerabilities
- Resolve jest issues due to upgrade to v24

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -39,7 +39,7 @@
     "pg": "7.6.0",
     "request": "2.88.0",
     "request-promise-native": "1.0.5",
-    "sequelize": "4.41.0",
+    "sequelize": "^6.0.0",
```
Comment (Mark-Wood) [inline package.json]: The carat symbol signifies that newer patch releases will be used and therefore should be removed if we are formally testing against a specific dependency version. Replace this with the exact version you have run this code with (you can check the dependency's package.json file in the node_modules directory.

File: package.json [hunk 2]
```diff
@@ -49,8 +49,8 @@
     "eslint": "5.8.0",
     "eslint-config-prettier": "3.1.0",
     "eslint-plugin-prettier": "3.0.0",
-    "jest": "23.6.0",
-    "jest-junit": "5.2.0",
+    "jest": "^24.8.0",
+    "jest-junit": "^7.0.0",
```
Comment (Mark-Wood) [inline package.json]: Same as above regarding carat.

File: package.json [hunk 3]
```diff
@@ -49,8 +49,8 @@
     "eslint": "5.8.0",
     "eslint-config-prettier": "3.1.0",
     "eslint-plugin-prettier": "3.0.0",
-    "jest": "23.6.0",
-    "jest-junit": "5.2.0",
+    "jest": "^24.8.0",
```
Comment (Mark-Wood) [inline package.json]: Same as above regarding carat.

File: package.json (hunk 1 above)
Comment (michalrak) [reply package.json]: Locked the versions now.

**Pull request merged**